### DISCUSSION TITLE
기말 최종 과제

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#49] [Show and tell]
# 기말 Data Visualization 프로젝트 

## 1. 프로젝트 블로그 링크
아래 내용 필수 포함

**Comment by SeyounJung01** [comment]
정세윤
https://blog.naver.com/seyounjj2001/222600454270
http://uosgenerative.site/seyoun/
https://editor.p5js.org/jsypaul2001/sketches/X4pHZgaEk

**Comment by jwejwepark** [comment]
박주혜
https://blog.naver.com/glqkfl0726/222600491208
http://uosgenerative.site/juhye/
https://editor.p5js.org/juhye/sketches/9_JR4efXu

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222600492661
http://uosgenerative.site/hong/
https://editor.p5js.org/tnekf1012/sketches/lH4LC4WwO

**Comment by imyeeunok** [comment]
https://blog.naver.com/010312kye/222600496927
http://uosgenerative.site/kimyeng/
https://editor.p5js.org/kimyeng/sketches/-qwrqR0fE

**Comment by Joeun410** [comment]
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222600509377&categoryNo=29&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=4208

https://editor.p5js.org/joeun10/sketches/hvIl5UZvh

http://uosgenerative.site/joeun/

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222600592021
http://uosgenerative.site/hyunjung/
https://editor.p5js.org/KimHyunJung/sketches/4QYwSQa7i
